website/content/docs/connect/connect-internals.mdx

```diff
@@ -11,21 +11,20 @@ This topic describes how many of the core features of Consul's service mesh func
 It is not a prerequisite,
 but this information will help you understand how Consul service mesh behaves in more complex scenarios.
 
-Consul Connect is the component shipped with Consul that enables service mesh functionality. The terms _Consul Connect_ and _Consul service mesh_ are used interchangeably throughout this documentation.
-
+Consul Service Mesh is the component shipped with Consul that enables service mesh functionality.
 To try service mesh locally, complete the [Getting Started with Consul service
 mesh](https://learn.hashicorp.com/tutorials/consul/service-mesh?utm_source=docs)
 tutorial.
 
 ## Mutual Transport Layer Security (mTLS)
 
-The core of Connect is based on [mutual TLS](https://en.wikipedia.org/wiki/Mutual_authentication).
+The core of Consul Service Mesh is based on [mutual TLS](https://en.wikipedia.org/wiki/Mutual_authentication).
```

Comment:
```suggestion
The core of Consul service mesh is based on [mutual TLS](https://en.wikipedia.org/wiki/Mutual_authentication).
```
I believe per the style guide, service mesh does not need to be consistently capitalized.